src/CoreMedia/CMSync.cs

```diff
@@ -147,7 +171,52 @@ public CMTimebase (CMTimebase masterTimebase)
 
 			CFObject.CFRetain (Handle);
 		}
+		
+#if !NET
+		[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0)]
+#else
+		[SupportedOSPlatform ("ios15.0"), SupportedOSPlatform ("tvos15.0"), SupportedOSPlatform ("macos12.0")]
+#endif
+		[DllImport(Constants.CoreMediaLibrary)]
+		static extern CMTimebaseError CMTimebaseCreateWithSourceClock (/* [NullAllowed] CFAllocatorRef */ IntPtr allocator, /* CMClock */ IntPtr sourceClock, /* CMTimebase */ out IntPtr  timebaseOut);
+
+#if !NET
+		[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0)]
+#else
+		[SupportedOSPlatform ("ios15.0"), SupportedOSPlatform ("tvos15.0"), SupportedOSPlatform ("macos12.0")]
+#endif
+		public CMTimebase (CFAllocator? allocator, CMClock sourceClock)
+		{
+			if (sourceClock == null)
```

Comment:
Very minor, no need to change if nothing else has to change:

```suggestion
			if (sourceClock is null)
```